test/integration/src/test/java/io/pravega/test/integration/endtoendtest/EndToEndReaderGroupTest.java

```diff
@@ -153,6 +156,92 @@ public void testDeleteReaderGroup() throws Exception {
         assertEquals("data1", eventRead.getEvent());
     }
 
+    @Test
```

Comment:
For these tests, can we also add an assertion that advancing the reader group will correctly result in data being truncated?

Reply:
Added a test for the case.